config/metadata-processor/src/test/java/io/helidon/config/metadata/processor/ConfigMetadataProcessorTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.config.metadata.processor;
+
+import org.junit.jupiter.api.Test;
+
+class ConfigMetadataProcessorTest {
+    @Test
+    void testToConfigKey() {
+//        ConfigMetadataProcessor p = new ConfigMetadataProcessor();
```

Comment:
What's the reason for this to be commented-out ?
You could use the `org.junit.jupiter.api.Disabled` annotation with some text to specify the reason.

Reply:
Forgot to uncomment